src/routes/Router.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const DiscoverHandler = require('../handlers/DiscoverHandler');
+
+module.exports = class Router {
+  /**
+   * Configures the routes for the application
+   * The express app reference is passed as the first argument
+   * The handlerRegistry is a module with according handler references
+   *
+   * @param {*} app
+   * @param {*} handlerRegistry
+   * @return {Array}
+   */
+  config(app, handlerRegistry) {
+    const routes = [];
+
+    // Index
+
+    const indexPath = '/';
+    app.get(indexPath, (req, res) => {
+      const result = {
+        message: 'oh, hi there',
+      };
+      res.json(result);
+    });
+
+    routes.push({
+      path: `get: ${indexPath}`,
+    });
+
+    // Heartbeat
+
+    const heartbeatPath = '/heartbeat/';
+    app.get(heartbeatPath, (req, res) => {
+      const result = {
+        message: 'tum-tum',
+      };
+      res.json(result);
+    });
+
+    routes.push({
+      path: `get: ${heartbeatPath}`,
+    });
+
+
+    /**
+     * The handlers for specific endpoints go here
+     * TODO: maybe we could use a config file like a sls.yml but our own?
```

Comment:
maybe dotenv npm package might be usefull in combination with a js file like this one
https://github.com/studyportals/StudentEvents/blob/develop/config/serverless/serverless.config.js

Reply:
Oh, that's awesome! Yeah, we could think of different options, but indeed could be useful. 
And then we could have the config first routing maybe (question is whether we even need and want it?)